tensorflow/contrib/metrics/python/ops/metric_ops.py

```diff
@@ -3297,9 +3299,120 @@ def count(values,
     return count_, update_op
 
 
+def cohen_kappa(labels, predictions, num_classes, weights=None,
```

Comment:
Can you make this argument optional, similar to the confusion_matrix method? Thanks.

Reply:
Because tensorflow supports batch data, the maximum label might vary with each batch, hence I think it might be unsafe to let `num_classes` optional.